examples/auto_scaling_cluster/README.md

```diff
@@ -0,0 +1,3 @@
+# Auto Scaling Cluster
+
+This is an example of creating a auto-scale enabled single zone VPC Gen2 OpenShift cluster.
```

Comment:
This not what this example does.

The big question, what value does this example provide over the single zone example? The single zone example creates an identical cluster, with the add-on installed and enabled via the config map. This example has the add-on installed and disabled via the config map.

If (big if) it necessary to test installing the add-on in both enabled and disabled states, then a cleaner solution would be to expose the config map inputs via the variables (rather than as a local) in the single zone example.

Reply:
I created a separate example for auto-scaling  because it requires a data block to get cluster config and  kubernetes provider setup, without it is not possible to update the configmap.

`enableAutoscaling` was set to false because I was testing a use-case and left out to set it back to `true` before the PR.